UltiSnips/django.snippets

```diff
@@ -1,238 +1,367 @@
 priority -50
 
-# Generic Tags
-snippet %
-{% ${1} %}${2}
+# This files will define django snippets from sublime text djaneiro
+
+###########################################################################
+#                             FORMS SNIPPETS                              #
+###########################################################################
+snippet form "Form"
```

Comment:
Consider adding snippet options. Basically all of the snippets here should at least have the 'b' option. Also some of these snippets could profit from the new 'e' option (context). Which is awesome. you should read up on that :)

https://github.com/SirVer/ultisnips/blob/master/doc/UltiSnips.txt#L682